src/views/Upload.vue

```diff
@@ -31,7 +31,7 @@
         <ion-button fill="clear" @click="removeItem(product.sku)">{{ $t( "Remove" ) }}</ion-button>
       </ion-card>
       <ion-fab vertical="bottom" horizontal="end" slot="fixed">
-        <ion-fab-button @click="upload()">
+        <ion-fab-button @click="upload()" :disabled="Object.keys(uploadProducts).length === 0">
```

Comment:
As we have added the length check here, we can remove the same check from the `upload` method.